CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.17.0 (2021-04-22)
+
+- Updated the form control line height to fit descenders. [#261](https://github.com/blackbaud/skyux-forms/pull/261)
```

Comment:
```suggestion
- Updated the form control `line-height` to fit descenders. [#261](https://github.com/blackbaud/skyux-forms/pull/261)
```
No big deal if you don't think this change is necessary. I just think it might be a little clearer at a glance